modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -90,12 +142,13 @@
   [driver & {:keys [schema-names table-names]}]
   (sql/format {:select [[:c.column_name :name]
                         [:c.data_type :database-type]
-                        [[:- :c.ordinal_position 1] :database-position]
-                        [:c.schema_name :table-schema]
+                        [[:- :c.ordinal_position [:inline 1]] :database-position]
+                        [:c.table_schema :table-schema]
                         [:c.table_name :table-name]
                         [[:not= :pk.column_name nil] :pk?]
-                        [[:case [:not= :c.remarks ""] :c.remarks :else nil] :field-comment]]
-               :from [[:svv_all_columns :c]]
+                        [[:case [:not= :c.remarks [:inline ""]] :c.remarks :else nil] :field-comment]]
+               ;; svv_columns excludes columns from datashares, unlike svv_all_columns with includes them
+               :from [[:svv_columns :c]]
```

Comment:
This query was causing flakes, so I tried switching to use `svv_columns` instead of `svv_all_columns` and it seems to have made a difference. The difference between `svv_all_columns` and `svv_columns` is that `svv_all_columns` includes columns from other databases that the user has access to via redshift datashares. We don't support querying tables from redshift datashares in the product, and doing so will take some work, since there's no concept in the product of choosing database B after connecting to a database A.